util-images/access-tokens/example/example.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: access-tokens
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: access-tokens
+  namespace: access-tokens
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: access-tokens
+  namespace: access-tokens
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - get
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: access-tokens
+  namespace: access-tokens
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: access-tokens
+subjects:
+  - kind: ServiceAccount
+    name: access-tokens
+    namespace: access-tokens
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: access-tokens
```

Comment:
This is a single token, so access-token?

Reply:
Name for all resources is consistent and delivered from name of the utility (access-tokens).

If this name will be changed, then all other resources also need to have more specific names (like service-account-getter for a Role).

I will try to update all names into more descriptive form.